libs/components/layout/src/lib/modules/box/box.component.html

```diff
@@ -9,7 +9,9 @@
   }"
 >
   <div class="sky-box-header-content">
-    <ng-content select="sky-box-header" />
+    <span>
```

Comment:
This span was added so it could be used with `ViewChild`. Now that you're using `ContentChild`, can it be removed?